react-components/src/hooks/use3DModelName.tsx

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { useQuery, type UseQueryResult } from '@tanstack/react-query';
+import { useSDK } from '../components/RevealContainer/SDKProvider';
+
+export const use3DModelName = (ids: number[]): UseQueryResult<string[] | undefined, unknown> => {
+  const sdk = useSDK();
+
+  const queryResult = useQuery<string[] | undefined>(['cdf', '3d', 'model', ids], async () => {
+    const modelNames: string[] = await Promise.allSettled(
```

Comment:
When switching out to allSettled you have to transform/filter the values in the array, for example: 
![image](https://github.com/cognitedata/reveal/assets/70804363/e9bebdc0-d443-42f2-9c01-4e10857e68f8)

Reply:
So you filter out the rejected requests, and keep the non-rejected

Reply:
transformed the request to required results